pyplanet/apps/contrib/rankings/models/ranked_map.py

```diff
@@ -0,0 +1,10 @@
+from peewee import *
+from pyplanet.core.db import Model
+
+
+class RankedMap(Model):
+	id = IntegerField()
+	name = CharField(max_length=150)
+	uid = CharField(max_length=50)
+	author_login = CharField(max_length=100)
+	player_rank = IntegerField()
```

Comment:
Please add comments and explicitly define table name just like rank model.

Reply:
This doesn't (or at least shouldn't) result in a database table.
I need this model to execute the raw query for the best-/worst rank lists.

Will add the comments.